src/crud/crud_isochrone.py

```diff
@@ -1,191 +1,407 @@
+
+        start_time = time.time()
+
+        # Get network H3 cells
+        h3_3_grid = []
+        db = Database(settings.POSTGRES_DATABASE_URI)
+        try:
+            sql_get_h3_3_grid = f"""
+                WITH region AS (
+                    SELECT geom from {settings.NETWORK_REGION_TABLE}
+                )
+                SELECT g.h3_short FROM region r,
+                LATERAL temporal.fill_polygon_h3_3(r.geom) g;
+            """
+            for h3_index in db.select(sql_get_h3_3_grid):
+                if h3_index[0] in [8077]:
+                    h3_3_grid.append(h3_index[0])
+        except Exception as e:
+            print(e)
+        finally:
+            db.conn.close()
+
+        # Load segments into polars data frames
+        segments_df = {}
+        df_size = 0.0
+        try:
+            for h3_index in h3_3_grid:
+                print(f"Loading network for H3 cell {h3_index}.")
+
+                segments_df[h3_index] = pl.read_database_uri(
+                    query=f"""
+                        SELECT
+                            id, length_m, length_3857,
+                            class_, impedance_slope, impedance_slope_reverse,
+                            impedance_surface, CAST(coordinates_3857 AS TEXT) AS coordinates_3857,
+                            source, target, CAST(tags AS TEXT) AS tags, h3_3, h3_5
+                        FROM basic.segment
+                        WHERE h3_3 = {h3_index}
+                    """,
+                    uri=settings.POSTGRES_DATABASE_URI,
+                    schema_overrides=segment_schema,
+                )
+                segments_df[h3_index] = segments_df[h3_index].with_columns(
+                    pl.col("coordinates_3857").str.json_extract()
+                )
+                df_size += segments_df[h3_index].estimated_size("gb")
+        except Exception as e:
+            print(e)
+
+        print(f"Network load time: {round((time.time() - start_time) / 60, 1)} min.")
+        print(f"Network in-memory size: {round(df_size, 1)} GB.")
+
+        return segments_df
 
 
 class CRUDIsochrone:
-    def init_network():
-        #TODO: Add function here to read the data from database into polars.
-        # We should save the geofence_active_mobility in the goat database and read the network for the respective area in chunks of h3 cells.
-        # We could have the table dynamically defined in the config.py so we could ready other (smaller) extents for testing.
-        # From what we know at the moment we should have one polars df for each h3-res3 cell.
-        pass
+    def __init__(self):
+        self.valid_classes_walking = """
+            secondary,tertiary,residential,
+            livingStreet,trunk,unclassified,
+            parkingAisle,driveway,pedestrian,
+            footway,steps,track,bridleway,
+            unknown
+        """
 
+        self.valid_classes_bicycle = """
```

Comment:
I think it is better using an array here. It can be joined to a string using join operator.